tests/unit/src/test/scala/tests/DefinitionLspSuite.scala

```diff
@@ -182,7 +182,7 @@ class DefinitionLspSuite extends BaseLspSuite("definition") {
   // requires org.scalamacros:macroparadise and io.spire-match:kind-projector.
   // Navigation continues to mostly work, except for areas that have compilation
   // errors.
-  test("missing-compiler-plugin".flaky) {
+  test("missing-compiler-plugin") {
```

Comment:
The purpose of this test was to document a known limitation when the dependency classpath was missing symbols that are referenced in the library sources. It looks like we bumped up the cats dependency to a version that brings in simulacrum 🤔

Either way, we could remove this test instea d

Reply:
Ok cool! I removed it!